<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bg" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bg{
background-color:qlineargradient(spread:pad, x1:0, y1:0.551, x2:1, y2:0.511, stop:0 rgba(0, 0, 127, 255), stop:1 rgba(85, 170, 255, 255))}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>450</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Smart Traffic System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>450</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Register a new account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernamefield">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(143, 143, 143, 119);

font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordfield">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>460</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(143, 143, 143, 119);
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>430</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signupb">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>640</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
background-color: rgb(0, 0, 107);
color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordfield_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>550</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(143, 143, 143, 119);
color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>520</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Confirm Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>590</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="backb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailfield">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(143, 143, 143, 119);
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Downloads/istockphoto-1168877485-170667a.jpg</pixmap>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>220</y>
       <width>471</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:30px ;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>label_7</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>usernamefield</zorder>
    <zorder>passwordfield</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>signupb</zorder>
    <zorder>passwordfield_2</zorder>
    <zorder>label_5</zorder>
    <zorder>error</zorder>
    <zorder>backb</zorder>
    <zorder>emailfield</zorder>
    <zorder>label_6</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
